src/app/tests/TestICDManager.cpp

```diff
@@ -277,7 +277,7 @@ class TestICDManager
 
         // Set New durations for test case
         uint32_t oldActiveModeDuration_ms = icdConfigData.GetActiveModeDurationMs();
-        icdConfigData.SetModeDurations(0, icdConfigData.GetIdleModeDurationSec());
+        icdConfigData.SetModeDurations(MakeOptional(static_cast<uint32_t>(0)), NullOptional);
```

Comment:
Might have been more readable as `MakeOptional<uint32_t>(0)`.  Or not, hard to tell.